src/StatusQ/Controls/StatusInput.qml

```diff
@@ -0,0 +1,85 @@
+import QtQuick 2.14
+
+import StatusQ.Core 0.1
+import StatusQ.Core.Theme 0.1
+import StatusQ.Controls 0.1
+
+Rectangle {
+    id: root
+
+    implicitWidth: 480
+    height: (label.visible ? 
+                  label.anchors.topMargin +
+                  label.height :
+              charLimitLabel.visible ?
+                  charLimitLabel.anchors.topMargin +
+                  charLimitLabel.height :
+              0) +
+            statusBaseInput.anchors.topMargin +
+            statusBaseInput.height + 
+            (errorMessage.visible ? 
+                  errorMessage.anchors.topMargin +
+                  errorMessage.height :
+                  0) + 8
+
+    color: "transparent"
```

Comment:
if the color is always transparent, then why not make root an `Item`, not `Rectange`?

Reply:
Good catch. This is a leftover from debugging. Whoopsie